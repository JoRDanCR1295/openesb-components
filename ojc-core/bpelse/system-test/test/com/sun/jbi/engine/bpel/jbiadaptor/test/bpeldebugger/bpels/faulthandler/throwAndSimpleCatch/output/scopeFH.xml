<?xml version="1.0" encoding="UTF-8"?><jbi:message xmlns:msgns="http://www.mycomp.org/throwInsideScope/throwInsideScope" type="msgns:message1" version="1.0" xmlns:jbi="http://java.sun.com/xml/ns/jbi/wsdl-11-wrapper"><jbi:part>test scope level FHWithIn Scopes Fault Handler</jbi:part></jbi:message>
