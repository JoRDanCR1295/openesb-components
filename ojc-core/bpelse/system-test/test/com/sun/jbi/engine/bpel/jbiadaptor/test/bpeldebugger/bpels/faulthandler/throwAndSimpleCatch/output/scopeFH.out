======================================================================
======================================================================
======================================================================
======================================================================
======================================================================
INSERT: StateDBOImpl[BPEL ID: bpelse/bpeljbiadapter/build/test/classes/com/sun/jbi/engine/bpel/jbiadaptor/test/persistence/bpels/faulthandler/throwAndSimpleCatch/deployedFolder/throwInsideScope.bpel][ENGINE ID: DYNAMIC_ENGINE_ID]
INSERT: LastCheckPointDBOImpl[ACTIVITY ID: 2000004][OLD ACTIVITY ID: -1][TIMER VALUE: null][HAS FAULTED: false][FAULT NAME: null][PICK COMPOSITE ACT ID: 0][BRANCH INVOKE COUNTER: 0]
INSERT: VariableDBOImpl[ID: 1000000]
======================================================================
INSERT: FaultHandlerDataDBOImpl[Scope ID: 2000009][fault Name: {http://docs.oasis-open.org/wsbpel/2.0/process/executable}testFault1]
UPDATE: LastCheckPointDBOImpl[ACTIVITY ID: 2000013][OLD ACTIVITY ID: 2000004][TIMER VALUE: null][HAS FAULTED: false][FAULT NAME: null][PICK COMPOSITE ACT ID: 0][BRANCH INVOKE COUNTER: 0]
======================================================================
======================================================================
======================================================================
INSERT: VariableDBOImpl[ID: 1000001]
UPDATE: LastCheckPointDBOImpl[ACTIVITY ID: 2000012][OLD ACTIVITY ID: 2000013][TIMER VALUE: null][HAS FAULTED: false][FAULT NAME: null][PICK COMPOSITE ACT ID: 0][BRANCH INVOKE COUNTER: 0]
======================================================================
DELETE: StateDBOImpl[BPEL ID: bpelse/bpeljbiadapter/build/test/classes/com/sun/jbi/engine/bpel/jbiadaptor/test/persistence/bpels/faulthandler/throwAndSimpleCatch/deployedFolder/throwInsideScope.bpel][ENGINE ID: DYNAMIC_ENGINE_ID]
======================================================================
